datadog/resource_datadog_timeboard.go

```diff
@@ -13,15 +13,127 @@ import (
 )
 
 func resourceDatadogTimeboard() *schema.Resource {
+	apmOrLogQuery := &schema.Schema{
```

Comment:
Is this the same schema definition thats also in screenboards? If so do you think its worth it to find a way to refactor this so both reference the same definition?

Reply:
It's _almost_ the same, but some fields like `compute->interval` (that is an integer in timeboards, and a string in screenboards) are different.